.circleci/config.yml

```diff
@@ -0,0 +1,234 @@
+        - save_cache:
+           paths:
+             - ~/.conan
+           key: conan-cache-v1-linux--{{ checksum "conanfile.py" }}
+
+       # - run: 
+       #    name: Copy /opt/taraxa/existing files to project
+       #    command: cp -rp /opt/taraxa/cmake-docker-build-debug/* /root/project/cmake-docker-build-debug/
+#       - git/checkout-with-submodules
+#       - setup_remote_docker:
+#           docker_layer_caching: true
+        - run: 
+            name: Cmake
+            command: | 
+              cd $BUILD_OUTPUT_DIR &&
+              cmake -DCMAKE_BUILD_TYPE=Release \
+              -DTARAXA_STATIC_BUILD=ON \
+              -DTARAXAD_INSTALL_DIR=./bin_install \
+              -DTARAXAD_CONF_INSTALL_DIR=./bin_install \
+              ../ 
+        - run:
+            name: make clang-format
+            command: cd $BUILD_OUTPUT_DIR && make clang-format && make clang-format-check
+        - run: 
+           name: make
+           command: cd $BUILD_OUTPUT_DIR && make -j8 all 
+        - run: 
+           name: make install 
+           command: cd $BUILD_OUTPUT_DIR && make install 
+        - run: 
+           name: strip binary
+           command: cd $BUILD_OUTPUT_DIR && strip bin_install/taraxad && cp bin_install/taraxad bin_install/taraxad-x86_64
+        - run: 
+           name: Execute ctest 
+           command: cd $BUILD_OUTPUT_DIR/tests && ctest --output-on-failure
+        - store_artifacts:
+           path: /root/project/cmake-docker-build-debug/bin_install/taraxad
+        - persist_to_workspace:
+           root: cmake-docker-build-debug
+           paths: [ bin_install ]
+
+workflows:
+  build-and-release:
+    jobs:
+      - create-and-push-builder:
+          context: taraxa-node
+          filters:
+            branches: 
+              only: 
+               - develop
+               - cirleci
+      #- gcp-gcr/build-and-push-image:
+      #    executor: docker
+      #    image: taraxa-node/taraxa-builder
+      #    context: taraxa-node
+      #    setup-remote-docker: true
+      #    use-docker-layer-caching: true
+      - build-linux:
+          filters:
+            branches: 
+              ignore: /.*/
+            tags:  
+              only: /^v\d+\.\d+\.\d+$/
+          context: taraxa-node
+#         requires: [ gcp-gcr/build-and-push-image ]
+      - build-mac:
+          filters:
+            branches: 
+              ignore: /.*/
+            tags:  
+              only: /^v\d+\.\d+\.\d+$/
+          context: taraxa-node
+      - github-release/create:
+          context: taraxa-node
+          attach-workspace: true
+          workspace-root: cmake-docker-build-debug
+          path: cmake-docker-build-debug/bin_install/taraxad-x86_64
+          filters:
+            branches: 
+              ignore: /.*/
+            tags:  
+              only: /^v\d+\.\d+\.\d+$/
+          requires: 
+            - build-linux
+            - build-mac
```

Comment:
I think this only requires `build-linux`

Reply:
I think the goal is not to release anything if any build fail